digitalaidseattle/seattle-humane-app
# Feature/dillonoleary/removing-the-reports-page

## What type of PR is this? (check all applicable)


## Description

There are 4 pages on our app - Dashboard, Clients, Pets, and Reports. Reports seems like a duplicate because the Dashboard has the same information. I wonder if it's best to rename the dashboard too. I don't know if we want to merge this code, but I at least want to have it here so we can discuss it

## Related Tickets & Documents

None

## QA Instructions, Screenshots, Recordings

Removes the Reports tab. It isn't being used now anyways. We could just add it back later if we see a use for it

<img width="871" alt="Screenshot 2024-10-09 at 5 13 16 PM" src="https://github.com/user-attachments/assets/da95c80a-8936-4b32-8a60-b7b83cb3d38f">

## Added/updated tests?
_There should be 90% statement coverage and the coverage percent increases with each commit, unless a reviewer approves an exception._

      have not been included_

## Can reviewers merge this PR upon approval?
- [x] yes, reviewers can merge this PR - but you may not want to merge this. Kate will want to take a look
- [ ] no, the author of this PR needs to merge it

## [optional] Are there any post-deployment tasks we need to perform?
None

## [optional] Which Seattle Humane animal would you like to dedicate this PR to?
![alt_text](https://g.petango.com/photos/337/b23ab819-21cf-41b1-891d-720992abb8bd.jpg)
